in_toto/gpg/util.py

```diff
@@ -0,0 +1,222 @@
+"""
+<Module Name>
+  gpg/util.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 15, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  general-purpose utilities for binary data handling and pgp data parsing
+"""
+import struct
+import binascii
+import subprocess
+import shlex
+import re
+from distutils.version import StrictVersion
+
+import cryptography.hazmat.backends as backends
+import cryptography.hazmat.primitives.hashes as hashing
+import in_toto.gpg
+
+def get_mpi_length(data):
+  """
+  <Purpose>
+    parses an MPI (Multi-Precision Integer) buffer and returns the appropriate
+    length. This is mostly done to perform bitwise to byte-wise conversion.
+
+  <Arguments>
+    data: The MPI data
+
+  <Exceptions>
+    None
+
+  <Side Effects>
+    None
+
+  <Returns>
+    The length of the MPI contained at the beginning of this data buffer.
+  """
+  bitlength = int(struct.unpack(">H", data)[0])
+  return int((bitlength - 1)/8) + 1
+
+def hash_object(headers, algorithm, content):
```

Comment:
These functions are only separated from one another with a single blank line.  More than one blank is easier on the eyes.